internal/graph/check.go

```diff
@@ -1064,7 +1064,7 @@ func (c *LocalChecker) produceUsersets(ctx context.Context, usersetsChan chan us
 
 		usersetsMap[objectRel].Add(objectID)
 
-		if usersetsMap[objectRel].Size() > int(c.usersetBatchSize) {
+		if usersetsMap[objectRel].Size() >= int(c.usersetBatchSize) {
```

Comment:
**(question):** The title of this PR seems misleading, as this actually fixes a bug. Could we change it such that it will make it easier to add into the changelog as a bug fix? This seems important.